src/integration/resources/resources.go

```diff
@@ -89,18 +89,46 @@ func SetupCluster(
 	}
 
 	var (
-		aggDatabase = admin.DatabaseCreateRequest{
+		unaggDatabase = admin.DatabaseCreateRequest{
```

Comment:
ah so now by default test clusters will have two namespaces?

Reply:
Yeah, it already does. They were just being created incorrectly here previously. This also mirrors what we do with the existing integration tests.